mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -3285,6 +3303,37 @@ resource "google_compute_instance" "foobar" {
 `, instance)
 }
 
+func testAccComputeInstance_resourceManagerTags(instance string) string {
```

Comment:
We'll need to exercise both instances of `resource_manager_tags` in the test! We're testing network tags and not resource manager tags atm.